sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -966,14 +987,17 @@ public void run() {
                         ErrorAttachmentLog dumpAttachment = null;
                         Map.Entry<UUID, ErrorLogReport> unprocessedEntry = unprocessedIterator.next();
                         ErrorLogReport errorLogReport = unprocessedEntry.getValue();
+                        android.util.Log.d("DBUG", "1234 loop1 " + errorLogReport.report.getDevice().getWrapperSdkName());
```

Comment:
Please cleanup those

Reply:
I see outdated label without any updates. Most likely you reviewed old diff